cypress/utils/setup-utils.js

```diff
@@ -70,13 +77,182 @@ export const cleanUpRunOktaOrgs = (specRunVersionName) => {
   });
 };
 
-export const setupRunData = (specRunVersionName) => {
+export const setupOrgFacility = (specRunVersionName) => {
   let orgName = createOrgName(specRunVersionName);
   let facilityName = createFacilityName(specRunVersionName);
-  createAndVerifyOrganization(orgName)
+  return createAndVerifyOrganization(orgName)
```

Comment:
Added `return` here because spec 9 needs the facility id returned by the `addMockFacility` mutation